packages/style/package.json

```diff
@@ -17,7 +17,7 @@
     "test": ""
   },
   "devDependencies": {
-    "@jspsych/config": "^3.0.0",
+    "@jspsych/config": "^2.0.0",
```

Comment:
Just curious why you bumped this down to 2?

Reply:
v3 breaks builds.  I am going to fix this after we get the alpha release out.  I might not make it to the weekend.